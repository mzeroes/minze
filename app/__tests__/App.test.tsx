import React from 'react';
import renderer from 'react-test-renderer';
import { View } from 'react-native';

it('renders correctly with defaults', () => {
  const App = renderer.create(<View />).toJSON();
  expect(App).toMatchSnapshot();
});
