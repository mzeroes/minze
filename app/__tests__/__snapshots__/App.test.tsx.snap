// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly with defaults 1`] = `<View />`;
